tests/unit/validator/validationManager.spec.ts

```diff
@@ -387,4 +387,29 @@ describe('ValidationManager', () => {
       expect(refReason.outFailedReason).toBe(`classification is not a valid value.. Optional values: ${'NonValidClassification'}`);
     });
   });
+
+  describe('isPolygonValid', () => {
+    it('returns true when Polygon is valid', () => {
+      const footprint = createFootprint('Region');
+      const response = validationManager.isPolygonValid(footprint);
+      expect(response.isValid).toBe(true);
+    });
+
+    it('returns true when Polygon has BBOX', () => {
+      const footprint = createFootprint('Region');
+      footprint.bbox = [34.4077734887818, 31.454120054543704, 34.4780703169442, 31.5127352949048];
```

Comment:
This is hard-coded value, you can use faked generated value

Reply:
i want to use hard coded value as this is a test and it's need to be fixed

Reply:
Why? you can generate a valid bbox